### ISSUE TITLE
browser zoom doesn't work in watch mode

### ISSUE THREAD

**Issue by bo-ku-ra** [terrastruct/d2#943]
i cannot use v0.2.2!
when using the wait-mode on windows, the zoom function (25%-500%) of browsers (edge, chrome, firefox) is not available.
so i cannot edit the huge diagram sources. 
text too small!
i want to zoom in soon!

**Comment by alixander** [comment]
i don't understand this. can you rephrase or add more detail? @bo-ku-ra

**Comment by bo-ku-ra** [comment]
oh,  if only i had more english skills...

https://play.d2lang.com/?l=&script=TJLNaoUwEIX3eYrTwIV74SoScJOFT-JGm0kN1YxMtCkV373UFNrVwMf8wDfHGQvdOYMqwzOPg9TOaPUDOozCOZGo32pxKMCFtFro0tzH-zYR8sQzQWgZQkz4CCmMMz20OtVe5i4rFitJ4ljovwN12UnujZ54nYQXesIHIc-fffxiXuD3eEUDjikh8obM8n437a1qm-b2eDkuH_Xf662QO9V3AAAA__8%3D&

**Comment by alixander** [comment]
verified. thanks for reporting, we'll look into this
